playwright-e2e/dsm/pages/kitsInfo-pages/kitsSentPage.ts

```diff
@@ -49,6 +52,67 @@ export default class KitsSentPage {
     return await this.kitsTable.getData(columnName);
   }
 
+  /**
+   * Sorts the given column by clicking on it.
+   * Date type columns - first click gives you list starting with dates from long ago; second click starts with recent dates
+   * String type columns - first click gives you A -> Z list; second click gives you Z -> A list
+   * @param columnName name of the column to be sorted
+   */
+  public async sortColumn(opts: {
+    columnName: KitsColumnsEnum,
```

Comment:
It looks like these options are mutually exclusive, so to prevent the caller from making an error by setting too many of the fields, this method could be broken out into two methods like `sortAlphabetical(boolean startWithA)` and `sortByDate(boolean mostRecentFirst)`.